README.md

```diff
@@ -1,4 +1,51 @@
 # certbot
 
-TODO: Enter the cookbook description here.
+Usage
+-----
 
+The certbot cookbook manages SSL certificate generation and renewal within projects.
+
+Add `"recipe[certbot-cdh]"` to enable it.
+
+Remove any SSLs which are stored within data bags.
+
+Finally, include the following in the default attributes:
+
+```json
+"default_attributes": {
+  "certbot": {
+    "cert-owner": {
+      "email": "devops@inviqa.com"
+    }
+  },
+  "nginx": {
+    "shared_config": {
+      "<project-name": {
+        "protocols": ["http", "https"],
+        "includes_first": [
+          "certbot.conf"
+        ]
+      }
+    }
+  }
+}
+```
+
+Add the following cookbooks to the Berksfile:
+
+```text
+cookbook 'config-driven-helper', '~> 2.5'
+cookbook 'certbot', :github => 'inviqa/chef-certbot', :branch => 'master'
+cookbook 'certbot-cdh', :github => 'inviqa/chef-certbot-cdh', :branch => 'master'
```

Comment:
this is only for development, so would just be

```
cookbook 'certbot', '~> version'
```
no need to have the certbot-cdh in here as it's just certbot